Examples/CaseStudies/SwiftUICaseStudies/03-Navigation-NavigateAndLoad.swift

```diff
@@ -37,17 +37,18 @@ let navigateAndLoadReducer =
     with: Reducer<
       NavigateAndLoadState, NavigateAndLoadAction, NavigateAndLoadEnvironment
     > { state, action, environment in
+      struct CancelId: Hashable {}
       switch action {
       case .setNavigation(isActive: true):
         state.isNavigationActive = true
         return Effect(value: .setNavigationIsActiveDelayCompleted)
-          .delay(for: 1, scheduler: environment.mainQueue)
-          .eraseToEffect()
+          .deferred(for: 1, scheduler: environment.mainQueue)
```

Comment:
Any reason for this change? I think we can stick with Combine's `delay` to keep this example simple since it should behave the same.
```suggestion
          .delay(for: 1, scheduler: environment.mainQueue)
          .eraseToEffect()
```

Reply:
Just tried to save on the `.eraseToEffect()` line. )